### PR TITLE
Update Edge installation instructions

### PR DESCRIPTION
**Pull request by LyzardKing**
Add instructions to install the Edge extension on MacOS and Linux
Implementation depends on https://github.com/JabRef/jabref/pull/7056

### CODE REVIEW

**Review by tobiasdiez — CHANGES_REQUESTED** [review]
Thanks!
I think the github urls to the firefox json should also be updated, shouldn't they?

**Review by LyzardKing — COMMENTED** [review]
You're right.
It should be good now.

**Review by tobiasdiez — APPROVED** [review]
Thanks!

File: en/collect/jabref-browser-extension.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Most JabRef installations include the necessary files, so test the extension bef
 2. Install the JabRef browser extension: [Firefox](https://addons.mozilla.org/en-US/firefox/addon/jabref/?src=external-github), [Chrome](https://chrome.google.com/webstore/detail/jabref-browser-extension/bifehkofibaamoeaopjglfkddgkijdlh), [Vivaldi](https://chrome.google.com/webstore/detail/jabref-browser-extension/bifehkofibaamoeaopjglfkddgkijdlh)
 3. On Windows 7, please [upgrade Powershell](https://www.microsoft.com/en-us/download/details.aspx?id=54616).
 4. Download the following files and copy them to the same directory as `JabRef.exe`
-   * [jabref.json](https://raw.githubusercontent.com/JabRef/jabref/master/buildres/windows/jabref.json)
+   * [jabref-firefox.json](https://raw.githubusercontent.com/JabRef/jabref/master/buildres/windows/jabref-firefox.json)
```
Comment (koppor) [inline en/collect/jabref-browser-extension.md:29]: This is 404 - does some PR need to be merged?

Comment (LyzardKing) [reply en/collect/jabref-browser-extension.md:29]: Yes, that change is in https://github.com/JabRef/jabref/pull/7056

Comment (mlep) [reply en/collect/jabref-browser-extension.md:29]: @LyzardKing: Even after merging of JabRef/jabref#7056, the deadlink remains dead.

Comment (LyzardKing) [reply en/collect/jabref-browser-extension.md:29]: You're right. I changed the filename in https://github.com/JabRef/jabref/pull/7092

Comment (mlep) [reply en/collect/jabref-browser-extension.md:29]: Great!

**Pull request merged**